### PR TITLE
Account critical updates

### PR DESCRIPTION
**Pull request by itsalyse**
Updates the following critical-flagged articles in the Account section: 

Account Creation — https://docs.google.com/document/d/18nVandavarh4kGgnVDOgKSEiKTlTf6ydN81TRCsfSzA/edit

Changing Email —  https://docs.google.com/document/d/1lOoap6JV8aEEIDIloRqLQg6JJxO2zgI8IHHHK_tGSZI/edit#heading=h.39tpznrhhcqz

Changing Account Information — https://docs.google.com/document/d/1ee-oC8ei_kDIKWXBCdkHoy_IC4dKfYx7MVgu-UT63og/edit#heading=h.pbnpjzwksjqt

Suspended Account — https://docs.google.com/document/d/1N9gZjDx0ct6xH8yIsh1pZrIthbU70dysXIHCp2rPooI/edit#heading=h.keyjenghf0nm

Activity Tracking — https://docs.google.com/document/d/1FvH5m5HngDEujgXLemsOUmfMusXfbHNZYjE662nYUqk/edit#heading=h.e9izf6ajb8eq

Related to https://github.com/dnsimple/dnsimple-business/issues/1900

### CODE REVIEW

File: content/articles/activity-tracking.md [hunk 1]
```diff
@@ -16,35 +16,50 @@ categories:
...
+
+Locate the domain on your [Domain List](/articles/domain-list/). Click the domain name to access the **Overview** page. 
+
+Click **View** on the **Activity** card to view a detailed log of changes to the domain. 
 
 ![Activity Tracking at domain level](/files/activity-tracking-1.png)
-![Activity Tracking at domain level](/files/activity-tracking-2.png)
 
-## Activity tracking at the account level
+**new screenshot for tracking details**
```
Comment (itsalyse) [inline content/articles/activity-tracking.md:36]: https://drive.google.com/file/d/1aAaqyqwqPpnqk71MCIcMkopd3mpgdaWD/view?usp=share_link

File: content/articles/activity-tracking.md [hunk 2]
```diff
@@ -16,35 +16,50 @@ categories:
...
-Account-level activity tracking can be found under the account page. Scroll down to the activity section:
+Click **View** on the **Activity** card to view a detailed log of changes within your account, including:
 
-![Activity Tracking at account level](/files/activity-tracking-3.png)
-![Activity Tracking at account level](/files/activity-tracking-4.png)
+-Adjustments to account settings
+-Modifications affecting multiple domains
+-User actions and changes in permissions
+
+**new screenshots**
```
Comment (itsalyse) [inline content/articles/activity-tracking.md:50]: https://drive.google.com/file/d/1cIK2yXvxlJlDSi0zeVz0P4viAErJKhK1/view?usp=share_link

File: content/articles/activity-tracking.md [hunk 3]
```diff
@@ -16,35 +16,50 @@ categories:
...
-<info>
-Full audit history is available only on certain plans. [See the difference between plans](/articles/dnsimple-plans/).
-</info>
+The Events Filter displays custom event types within a chosen time frame, depending on whether you're interested in specific DNS changes, user actions, or time-specific events. Each entry provides detailed information, including:
+
+- What changed, along with contextual data (e.g. records content)
+- Which user triggered the event
+- When the event took place
+
+**new events screenshot**
```
Comment (itsalyse) [inline content/articles/activity-tracking.md:60]: https://drive.google.com/file/d/1BMBt1E2E_0H4HlDig1hE7hfjVEd1E6h5/view?usp=share_link

File: content/articles/changing-account-information.md [hunk 4]
```diff
@@ -16,26 +16,44 @@ categories:
...
 
-1. Change or set the information according to your needs:
+1. Change or set the information according to your needs.
+
+   You can edit your account name, email, country of operation, and change your answers to any of the security screening questions. 
 
    ![Account information form](/files/account-information-form.png)
 
    <info>
    DNSimple requests this information, in compliance with applicable laws and regulations, to guarantee the security of your account.
```
Comment (itsalyse) [inline content/articles/changing-account-information.md:50]: This is part of the Patriot Act, https://en.wikipedia.org/wiki/Know_your_customer and is just due diligence for this kind of business relationship. I don't think anyone's getting confused about whether or not they fall into these categories, but we can link the wikipedia article for the curious if that would be beneficial.

File: content/articles/changing-email.md [hunk 5]
```diff
@@ -31,20 +31,20 @@ categories:
 ## Changing the user email
 
 <div class="section-steps" markdown="1">
-1.  Log in to DNSimple with your user credentials.
-1.  Go to your [user page](https://dnsimple.com/user) by selecting the <label>Settings</label> link under your user menu.
+1.  Go to your [user page](https://dnsimple.com/user) by selecting the <label>User settings</label> link at the top right of the screen.
 
-    ![Settings menu](/files/access-user-settings.jpg)
+    **New screenshot**
```
Comment (itsalyse) [inline content/articles/changing-email.md:36]: https://drive.google.com/file/d/1FQOOExCUs7ckI6-RzdT60IoTRcr3Vgn-/view?usp=sharing

File: content/articles/changing-email.md [hunk 6]
```diff
@@ -31,20 +31,20 @@ categories:
...
+1.  Go to your [user page](https://dnsimple.com/user) by selecting the <label>User settings</label> link at the top right of the screen.
 
-    ![Settings menu](/files/access-user-settings.jpg)
+    **New screenshot** 
 
-1.  From the user page select the <label>Change Email Address</label> link.
+1.  In your user settings, select <label>Edit email</label>.
 
-    ![Email link](/files/user-email.png)
+    **New screenshot**
```
Comment (itsalyse) [inline content/articles/changing-email.md:40]: https://drive.google.com/file/d/1igzc6cRnr1yQLCVjdfs-JIksr9c5TO8F/view?usp=sharing

**Review by stephanoodle — APPROVED** [review]

**Review by dperrymorrow — APPROVED** [review]
I have added the screenshots from google drive

**Review by stephanoodle — APPROVED** [review]

**Review by Pemacaa — APPROVED** [review]

**Pull request merged**